Library/Formula/mlkit.rb

```diff
@@ -0,0 +1,43 @@
+class Mlkit < Formula
+  desc "Compiler for the Standard ML programming language"
+  homepage "https://melsman.github.io/mlkit"
+  url "https://github.com/melsman/mlkit/archive/mlkit-4.3.9.tar.gz"
+  sha256 "3c6adbeb9a85f7b3586d0961fd3b170ff31e09fa0ff12889b76b9ceb459059c4"
+
+  depends_on "autoconf" => :build
+  depends_on "mlton" => :build
+  depends_on "gmp"
+
+  def install
+    system "./autobuild"
+    system "./configure", "--prefix=#{prefix}"
+
+    # The ENV.permit_arch_flags specification is needed on 64-bit
+    # machines because the mlkit compiler generates 32-bit machine
+    # code whereas the mlton compiler generates 64-bit machine
+    # code. Because of this difference, the ENV.m64 and ENV.m32 flags
+    # are not sufficient for the formula as clang is used by both
+    # tools in a single makefile target. For the mlton-compilation of
+    # sml-code, no arch tags are used for the clang assembler
```

Comment:
Can you elaborate on this comment?

Den fredag den 1. april 2016 skrev ilovezfs <notifications@github.com>:

> In Library/Formula/mlkit.rb
> <https://github.com/Homebrew/homebrew/pull/49933#discussion_r58202676>:
>
> > +
> > +  depends_on "autoconf" => :build
> > +  depends_on "mlton" => :build
> > +  depends_on "gmp"
> > +
> > +  def install
> > +    system "./autobuild"
> > +    system "./configure", "--prefix=#{prefix}"
> > +
> > +    # The ENV.permit_arch_flags specification is needed on 64-bit
> > +    # machines because the mlkit compiler generates 32-bit machine
> > +    # code whereas the mlton compiler generates 64-bit machine
> > +    # code. Because of this difference, the ENV.m64 and ENV.m32 flags
> > +    # are not sufficient for the formula as clang is used by both
> > +    # tools in a single makefile target. For the mlton-compilation of
> > +    # sml-code, no arch tags are used for the clang assembler
>
> arch tags or arch flags?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/49933/files/b3e5bbd768c42d8b576ce685f5a8926f469733d8#r58202676>
>